packages/neon/neon/lib/src/widgets/list_view.dart

```diff
@@ -1,66 +1,79 @@
 import 'package:flutter/material.dart';
 import 'package:neon/src/widgets/exception.dart';
-import 'package:neon/src/widgets/linear_progress_indicator.dart';
 
-class NeonListView<T> extends StatelessWidget {
-  const NeonListView({
-    required this.items,
+// TODO: [Leptopoda] handle isLoading
```

Comment:
I think this is still not solved?

Reply:
I did revert it to the old behaviour and would like to change it with #800

Reply:
As far as I can tell `isLoading` is never used? So we don't have a visual indicator for loading if it wasn't triggered by the user pulling the refresh indicator.